chemprop/features/features_generators.py

```diff
@@ -50,84 +50,115 @@ def get_available_features_generators() -> List[str]:
 
 
 @register_features_generator('morgan')
-def morgan_binary_features_generator(mol: Molecule,
+def morgan_binary_features_generator(mol_data: Union[Molecule, List[List[Molecule]]],
```

Comment:
```suggestion
def morgan_binary_features_generator(mol_data: Union[Molecule, Sequence[Sequence[Molecule]]],
```
use as generic a type as possible (generally) in function signatures